app/controllers/public/loans_controller.rb

```diff
@@ -1,36 +1,25 @@
 class Public::LoansController < Public::PublicController
-  include WordpressEmbeddable
-  after_action :verify_authorized
-
   def index
-    params[:division] ||= Division.root.short_name
-    @params = { status: params[:status], pg: params[:pg], division: params[:division] }
-    @loans = policy_scope(Loan.filter_by_params(params).
-          includes(:organization, division: :parent).
-          page(params[:pg]).per(20).
-          order('signing_date DESC'))
-    authorize @loans
-
-    # Set last loan list URL for 'Back to Loan List' link
-    session[:loans_path] = request.fullpath
+    division = params[:division] || division_for_site(params[:site])
+    redirect_to controller: "divisions", action: "show", short_name: division
```

Comment:
Hooray for nice redirects!

Do we have spec coverage on these? It would be super easy to add if not (using request specs perhaps) and it would pay dividends I think. Redirects like this can be finicky...

Reply:
Also, I'm surprised these work currently given the `public` namespace is not specified in some way. Perhaps it figures it out b/c the current controller is also in `public`?